packages/guides/src/Nodes/InlineNode.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @link https://phpdoc.org
+ */
+
+namespace phpDocumentor\Guides\Nodes;
+
+use phpDocumentor\Guides\Nodes\InlineToken\InlineMarkupToken;
+use phpDocumentor\Guides\Nodes\InlineToken\PlainTextToken;
+
+/** @extends CompoundNode<InlineMarkupToken> */
+class InlineNode extends CompoundNode
```

Comment:
Should we keep this? Or could we remove it since the new nodes can be added directly to their parent? Like

Paragraph
  InlineNode
     LiteralNode
     HyperLinkNode

Could be simplified to:
Paragraph
     LiteralNode
     HyperLinkNode

Reply:
The rendering of the InlineNode takes care of  for example not applying spaces etc and it is resused in all Places where Text with Markup is used. The Paragraph adds a <p> tag around a text. So they are different things in my eyes.